plugin/completion/lib_complete.py

```diff
@@ -81,22 +82,22 @@ def __init__(self, clang_binary):
             # bindings for python 3
             log.debug(
                 " using bundled cindex: %s", cindex_module_name)
-            cindex = importlib.import_module(cindex_module_name)
+            Completer.cindex = importlib.import_module(cindex_module_name)
```

Comment:
Can we just store `[Completer.cindex.CursorKind.DESTRUCTOR, Completer.cindex.CursorKind.CLASS_DECL]` as members of this class and not put cindex as a class variable? I think it would be better this way.